packages/devtools/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/flutter/devtools
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: '>=2.6.0 <3.0.0'
```

Comment:
we should keep this version and the version in the server at 2.3 or maybe 2.5 unless there is a strong reason not to. That way people with slightly old versions of Flutter can still use the latest version of DevTools. By contrast, we precompile devtools_app so using a tip of trunk sdk does not impact our users.

Reply:
Done. Now the only bump in this CL is to `devtools_app/pubspec.yaml`

Reply:
maybe add a comment in the other pubspec.yaml explaining why we keep intentionally diverge the sdk versions?